resources/js/views/settings/Application.vue

```diff
@@ -0,0 +1,468 @@
+          required
+          :disabled='isBusy || viewOnly'
+          v-if="roomLimitMode === 'custom'">
+        </b-form-input>
+
+        <template slot='invalid-feedback'>
+          <div v-html="fieldError('room_limit')"></div>
+        </template>
+      </b-form-group>
+
+      <!--Pagination page size settings-->
+      <b-form-group
+        label-class="font-weight-bold"
+        class="mb-4"
+        label-for="application-pagination-page-size-input"
+        :description="$t('settings.application.paginationPageSize.description')"
+        :state='fieldState("pagination_page_size")'
+        :label="$t('settings.application.paginationPageSize.title')"
+      >
+        <b-form-input id="application-pagination-page-size-input"
+                      v-model="settings.pagination_page_size"
+                      type="number"
+                      :disabled="isBusy || viewOnly"
+                      min="1"
+                      max="100"
+                      required
+                      :state='fieldState("pagination_page_size")'
+        >
+        </b-form-input>
+
+        <template slot='invalid-feedback'>
+          <div v-html="fieldError('pagination_page_size')"></div>
+        </template>
+      </b-form-group>
+
+      <!--Own rooms pagination page size settings-->
+      <b-form-group
+        label-class="font-weight-bold"
+        class="mb-4"
+        label-for="application-pagination-own-room-page-size-input"
+        :description="$t('settings.application.ownRoomsPaginationPageSize.description')"
+        :state='fieldState("own_rooms_pagination_page_size")'
+        :label="$t('settings.application.ownRoomsPaginationPageSize.title')"
+      >
+        <b-form-input id="application-pagination-own-room-page-size-input"
+                      v-model="settings.own_rooms_pagination_page_size"
+                      type="number"
+                      min="1"
+                      max="25"
+                      required
+                      :disabled="isBusy || viewOnly"
+                      :state='fieldState("own_rooms_pagination_page_size")'
+        >
+        </b-form-input>
+
+        <template slot='invalid-feedback'>
+          <div v-html="fieldError('own_rooms_pagination_page_size')"></div>
+        </template>
+      </b-form-group>
+
+    <hr>
+    <b-button id="application-save-button"
+              class="float-right"
+              variant="success"
+              type="submit"
+              v-if="!viewOnly"
+              :disabled="isBusy">
+      <span><i class="fas fa-save mr-2"></i>{{ $t('app.save') }}</span>
+    </b-button>
+
+    </b-form>
+    </b-overlay>
+    </b-container>
+</template>
+
+<script>
+import Base from '../../api/base';
+import FieldErrors from '../../mixins/FieldErrors';
+import env from '../../env';
+import PermissionService from '../../services/PermissionService';
+
+export default {
+  mixins: [FieldErrors],
+
+  data () {
+    return {
+      loaded: false,
```

Comment:
Why the extra loaded if there is a isBusy?

Reply:
to hide the empty fields on the inital page load